publicodes/package.json

```
@@ -2,13 +2,21 @@
 	"name": "publicodes",
 	"version": "1.0.0",
 	"description": "A declarative language for encoding public algorithm",
-	"main": "source/index.ts",
+	"main": "dist/index.js",
+	"types": "dist/types/index.d.ts",
 	"repository": {
 		"type": "git",
 		"url": "https://github.com/betagouv/mon-entreprise.git",
 		"directory": "publicodes"
 	},
+	"bugs": "https://github.com/betagouv/publicodes",
```

Vu que le code est ici, on pourrait rediriger les bugs vers le monorepo aussi

```suggestion
	"bugs": "https://github.com/betagouv/mon-entreprise/issues/new",
```

Et éventuellement configurer un template "Publicode" qui demande la numéro de version (exemple de templates sur un repo GitHub : https://github.com/zeit/next.js/issues/new/choose)